notifier.go

```diff
@@ -75,6 +87,9 @@ func NewNotifier(projectId int64, projectKey string) *Notifier {
 	if wd, err := os.Getwd(); err == nil {
 		n.context["rootDirectory"] = wd
 	}
+	for i := 0; i < 10; i++ {
+		go n.worker()
```

Comment:
It is ignored, see https://github.com/airbrake/gobrake/blob/feature/notify-and-panic/notifier.go#L179-L184